Assets/AdVdGlyphRecognition/Scripts/GlyphDrawInput.cs

```diff
@@ -103,7 +103,8 @@ public enum Matching_Method{ None=-1, SqrDistanceDTWMatchingMethod, SqrDTWMatchi
 				matchingMethod=value;
 			}
 		}
-		MatchingMethod method;
+		// This is bad, fix this
```

Comment:
I need to change this, I made this public for the time being as they way the library is set up I can't seem to find a way to easily match a glyph on command. They actual public variable that can be accessed just tells you what matching method is going to be used. The proper way to do this would be to instantiate my own MatchingMethod class in the GlyphRecognition script